test/integration/pack/test_pack.py

```diff
@@ -631,6 +632,9 @@ def test_pack_rpm(tt_cmd, tmpdir):
 
 @pytest.mark.slow
 def test_pack_rpm_use_docker(tt_cmd, tmpdir):
+    if platform.system() == "Darwin":
+        pytest.skip("/set platform is unsupported")
```

Comment:
Why do we skip pack --use-docker for OSX? It should work AFAIK. @vr009

Reply:
Removed